contrib/babelfishpg_tsql/src/rolecmds.c

```diff
@@ -2430,3 +2455,25 @@ remove_createrole_from_logins(PG_FUNCTION_ARGS)
 	table_close(rel, AccessShareLock);
 	PG_RETURN_INT32(0);
 }
+
+PG_FUNCTION_INFO_V1(bbf_is_member_of_role_nosuper);
+Datum
+bbf_is_member_of_role_nosuper(PG_FUNCTION_ARGS)
+{
+	Oid	member, role;
+	char	*rolename;
+	char	*dc_role;
+	bool	result;
+
+	if (PG_ARGISNULL(0) || PG_ARGISNULL(1))
+		PG_RETURN_NULL();
+
+	member = PG_GETARG_OID(0);
+	rolename = text_to_cstring(PG_GETARG_TEXT_P(1));
+	dc_role = downcase_identifier(rolename, strlen(rolename), false, false);
+	role = get_role_oid(dc_role, false);
```

Comment:
Can we pass the oid for rolename as well? To avoid down-casing and fetching the oid here.